.github/workflows/checks.yaml

```diff
@@ -49,6 +49,9 @@ jobs:
       - name: Run integration tests
         shell: bash
         run: pytest tests/test_integration
+      - name: Run unit tests
+        shell: bash
+        run: pytest tests/unitTest/pro_tes/middleware
       - name: Tear down app
         run: docker-compose down
```

Comment:
Is this really not indented? Looks like that to me on my review screen on the phone...

Reply:
This is well Indented.